tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -18,11 +12,114 @@
     JOB_STATE_RUNNING,
 )
 
-from .ensemble_evaluator_utils import AutorunTestEnsemble, send_dispatch_event
+from .ensemble_evaluator_utils import send_dispatch_event
+
+
+def test_new_monitor_can_pick_up_where_we_left_off(evaluator):
+    evaluator._start_running()
+    token = evaluator._config.token
+    cert = evaluator._config.cert
+    url = evaluator._config.url
+
+    config_info = evaluator._config.get_connection_info()
+    with Monitor(config_info) as monitor:
+        events = monitor.track()
+        # first snapshot before any event occurs
+        snapshot_event = next(events)
+        snapshot = Snapshot(snapshot_event.data)
+        assert snapshot.status == ENSEMBLE_STATE_UNKNOWN
+        # two dispatchers connect
+        with Client(
+            url + "/dispatch",
+            cert=cert,
+            token=token,
+            max_retries=1,
+            timeout_multiplier=1,
+        ) as dispatch1, Client(
+            url + "/dispatch",
+            cert=cert,
+            token=token,
+            max_retries=1,
+            timeout_multiplier=1,
+        ) as dispatch2:
+            # first dispatcher informs that job 0 is running
+            send_dispatch_event(
+                dispatch1,
+                identifiers.EVTYPE_FM_JOB_RUNNING,
+                f"/ert/ensemble/{evaluator.ensemble.id_}/real/0/step/0/job/0",
+                "event1",
+                {"current_memory_usage": 1000},
+            )
+
+            # second dispatcher informs that job 0 is running
+            send_dispatch_event(
+                dispatch2,
+                identifiers.EVTYPE_FM_JOB_RUNNING,
+                f"/ert/ensemble/{evaluator.ensemble.id_}/real/1/step/0/job/0",
+                "event1",
+                {"current_memory_usage": 1000},
+            )
+            send_dispatch_event(
```

Comment:
I guess we miss here: # second dispatcher informs that job 1 is running

Reply:
thank you, fixed in 345d9a44e209df99ca069c722964144a747caf7c